[2025-06-05T07:33:22.446+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-05T07:33:22.468+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: excel_to_postgres_test_data_dag.load_excel_data_to_postgres manual__2025-06-05T07:33:17.762156+00:00 [queued]>
[2025-06-05T07:33:22.475+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: excel_to_postgres_test_data_dag.load_excel_data_to_postgres manual__2025-06-05T07:33:17.762156+00:00 [queued]>
[2025-06-05T07:33:22.476+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-06-05T07:33:22.487+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_excel_data_to_postgres> on 2025-06-05 07:33:17.762156+00:00
[2025-06-05T07:33:22.490+0000] {standard_task_runner.py:63} INFO - Started process 36 to run task
[2025-06-05T07:33:22.493+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'excel_to_postgres_test_data_dag', 'load_excel_data_to_postgres', 'manual__2025-06-05T07:33:17.762156+00:00', '--job-id', '164', '--raw', '--subdir', 'DAGS_FOLDER/excel_to_postgres_test_data_dag.py', '--cfg-path', '/tmp/tmp5ovc59q_']
[2025-06-05T07:33:22.495+0000] {standard_task_runner.py:91} INFO - Job 164: Subtask load_excel_data_to_postgres
[2025-06-05T07:33:22.534+0000] {task_command.py:426} INFO - Running <TaskInstance: excel_to_postgres_test_data_dag.load_excel_data_to_postgres manual__2025-06-05T07:33:17.762156+00:00 [running]> on host e40159cd4691
[2025-06-05T07:33:22.616+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='excel_to_postgres_test_data_dag' AIRFLOW_CTX_TASK_ID='load_excel_data_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-06-05T07:33:17.762156+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-05T07:33:17.762156+00:00'
[2025-06-05T07:33:22.617+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-05T07:33:22.788+0000] {base.py:84} INFO - Using connection ID 'con_datalake' for task execution.
[2025-06-05T07:33:22.878+0000] {logging_mixin.py:188} INFO - Đã chèn 52 dòng dữ liệu vào test_***.test_data_tara.
[2025-06-05T07:33:22.878+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-06-05T07:33:22.879+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-05T07:33:22.887+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=excel_to_postgres_test_data_dag, task_id=load_excel_data_to_postgres, run_id=manual__2025-06-05T07:33:17.762156+00:00, execution_date=20250605T073317, start_date=20250605T073322, end_date=20250605T073322
[2025-06-05T07:33:22.906+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-05T07:33:22.921+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-05T07:33:22.924+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
